client/source/DelphiLint.SettingsFrame.pas

```diff
@@ -61,18 +64,37 @@   TLintSettingsFrame = class(TFrame)
     VersionRefreshButton: TButton;
     Panel2: TPanel;
     Panel3: TPanel;
+    Label5: TLabel;
+    StandaloneRulesRadioGroup: TRadioGroup;
+    StandaloneRulesListBox: TCheckListBox;
+    Panel1: TPanel;
+    StandaloneRulesPopupMenu: TPopupMenu;
+    EnableAll1: TMenuItem;
+    DisableAll1: TMenuItem;
```

Comment:
Can we rename it to `DisableAll` so that we don't have this stray `1` in the field and method name?